design/src/SUMMARY.md

```diff
@@ -1,10 +1,11 @@
 # Summary
 
 - [Draft Steam Page](./steam-page.md)
+- [Glossary](./glossary.md)
 - [High Level Analysis](./high-level/index.md)
   - [Game Thesis](./high-level/game-thesis.md)
   - [Genre Analysis](./high-level/genre-analysis.md)
   - [Genre Mechanics](./high-level/genre-mechanics.md)
-  - [Mapping genre mechanics to *Emergence*](high-level/mechanical-translation.md)
+  - [Mapping genre mechanics to *Emergence*](./high-level/mechanical-translation.md)
```

Comment:
Wasn't this fixed in the other PR already?

Reply:
This is due to lack of rebasing.